app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -34,5 +34,11 @@ def t(key, interpolations = {})
     end
     helper HelperOverride
 
+    # Adds the 'My Works' breadcrumb; we only want this for logged-in users
+    # overrides https://github.com/samvera/sufia/blob/v7.3.1/app/controllers/concerns/sufia/works_controller_behavior.rb#L93
+    def add_breadcrumb_for_controller
+      super if current_ability.current_user.staff?
+    end
```

Comment:
@jrochkind we could change the `staff?` method I linked to above to `logged_in?` and then alias `staff?` back to it, changing this to use `logged_in?` would that work?

That method uses a check for belonging in the group `registered`. You can also check `persisted?` on the user; the dummy object returns `false`.